### PR TITLE
[Package Add] Add zotero

### PR DESCRIPTION
**Pull request by JonBoyleCoding**

### CODE REVIEW

File: packages/zotero/zotero.pacscript [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+}
+
+install() {
+
+  sudo install -D zotero.desktop -t /usr/src/pacstall/${name}/usr/share/applications
+
+  installdir=/usr/src/pacstall/${name}/opt/zotero
+
+  mkdir -p "${installdir}"
+  sudo cp -r * "${installdir}"
```
Comment (wizard-28) [inline packages/zotero/zotero.pacscript:28]:
```suggestion
  sudo cp -r ./ "${installdir}"
```

**Review by wizard-28 — CHANGES_REQUESTED** [review]

File: packages/zotero/zotero.pacscript [hunk 2]
```diff
@@ -0,0 +1,29 @@
+name="zotero"
+version="5.0.96.2"
+url="https://download.zotero.org/client/release/${version}/Zotero-${version}_linux-x86_64.tar.bz2"
+description="Your personal research assistant"
+hash="653faac4d2e3746dbd0e1c95d095ed8031c38b68183f22ab6708093aba392082"
+maintainer="Falcon Programmer <programmerfalcon@gmail.com>"
```
Comment (wizard-28) [inline packages/zotero/zotero.pacscript:6]: Who is this `Falcon Programmer`? This field should point to @JonBoyleCoding's details

Comment (wizard-28) [reply packages/zotero/zotero.pacscript:6]:
```suggestion
maintainer="JonBoyleCoding <programmerfalcon@gmail.com>"
```
Changing it to your GH username will help users find you.

File: packages/zotero/zotero.pacscript [hunk 3]
```diff
@@ -0,0 +1,29 @@
+name="zotero"
+version="5.0.96.2"
+url="https://download.zotero.org/client/release/${version}/Zotero-${version}_linux-x86_64.tar.bz2"
+description="Your personal research assistant"
+hash="653faac4d2e3746dbd0e1c95d095ed8031c38b68183f22ab6708093aba392082"
+maintainer="JonBoyleCoding <programmerfalcon@gmail.com>"
+
+tarfile="Zotero-${version}_linux-x86_64.tar.bz2"
```
Comment (wizard-28) [inline packages/zotero/zotero.pacscript:8]:
```suggestion

```
Its not used anywhere, remove it

**Review by wizard-28 — CHANGES_REQUESTED** [review]

File: packages/zotero/zotero.pacscript (hunk 3 above)
Comment (JonBoyleCoding) [reply packages/zotero/zotero.pacscript:8]: Ah right yes, I forgot to remove it.

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Nice, this will work. 

Now all is left is to edit the [`packagelist`](https://github.com/JonBoyleCoding/pacstall-programs/blob/zotero/packagelist) file, and add this app's name there alphabetically

And you have been invited to the Pacstall Organization, so we can @mention, add you as a reviewer etc, please accept the invitation.

**Review by wizard-28 — APPROVED** [review]
Merging

**Pull request merged**